x-pack/plugins/observability_onboarding/public/index.ts

```diff
@@ -5,17 +5,38 @@
  * 2.0.
  */
 
-import { PluginInitializer, PluginInitializerContext } from '@kbn/core/public';
+import {
+  ApplicationStart,
+  HttpStart,
+  PluginInitializer,
+  PluginInitializerContext,
+} from '@kbn/core/public';
 import {
   ObservabilityOnboardingPlugin,
   ObservabilityOnboardingPluginSetup,
   ObservabilityOnboardingPluginStart,
 } from './plugin';
 
+export interface ConfigSchema {
+  ui: {
+    enabled: boolean;
+  };
+  serverless: {
+    enabled: boolean;
+  };
+}
+
+export interface ObservabilityOnboardingAppServices {
+  application: ApplicationStart;
+  http: HttpStart;
+  config: ConfigSchema;
+}
+
```

Comment:
This part is not directly related to this PR, but i realised while working on this PR that the `observability_onboarding` plugin currently does not exposes config at all, 

My initial implementation required that, though i refactored the code and its not required anymore. But it's good to have the config available to the frontend.